src/main/java/com/hyperwallet/clientsdk/model/HyperwalletVerificationDocument.java

```diff
@@ -24,6 +22,8 @@
 
     private String country;
 
+    private List<HyperwalletDocumentRejectReason> reasons;
+
```

Comment:
@hrandhawa-hw should we add createdOn also here?

Reply:
I think createdOn should be there too. @simrandsingh do other objects have a field for createdOn?

Reply:
yes, many classes have it like: HyperwalletUser, HyperwalletBankAccount, HyperwalletBankCard,  and etc

Reply:
I'll add createdOn in HyperwalletVerificationDocument.java.

Reply:
I'll add createdOn in HyperwalletVerificationDocument.java.